openedx/features/smart_referral/tests/test_views.py

```diff
@@ -0,0 +1,69 @@
+import mock
+from ddt import data, ddt
+from django.urls import reverse
+from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_403_FORBIDDEN
+from rest_framework.test import APITestCase
+
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+
+from .factories import SmartReferralFactory
+
+
+@ddt
+class SmartReferralInvitationAPIViewTest(APITestCase):
+
+    def setUp(self):
+        super(SmartReferralInvitationAPIViewTest, self).setUp()
+        self.user = UserFactory()
+        self.user.set_password('password')
+        self.user.save()
+        self.client.login(username=self.user.username, password='password')
+
+    @mock.patch('openedx.features.smart_referral.views.task_referral_and_toolkit_emails')
```

Comment:
Any specific reason to use views path to mock task, if not then let's please mock it from task `@mock.patch('openedx.features.smart_referral.tasks.task_referral_and_toolkit_emails')`

Reply:
For mocking, we use the path where the function is used instead of the path where it is created

Reply:
Thanks! I was not aware of it.